<launch>

	<arg name="item_id" default="unnamed" />
	<arg name="planner_id" default="GridBased" />

  <node pkg="asb_task" exec="global_plan_viz.py" name="global_plan_viz">
    <param name="source_task_plan_file_path" value="~/w/agrosensebot_ws/src/AgroSenseBot/asb_task/config/spraying_task_plans/cornaredo.yaml" />  <!-- kind of hardcoded, but there is no way to find the source directory of a package with ros2 launch and rclpy -->
    <param name="planner_id" value="$(var planner_id)" />
    <param name="task_plan_item_id" value="$(var item_id)" />
  </node>

</launch>
